### ISSUE TITLE
Wrong error msg in "Submit" tab when NO submit role is connected to a user

### ISSUE THREAD

**Issue by sbujam** [inveniosoftware/invenio#1414]
Originally on 2013-03-19

When a logged-in user has NO sumbit permissions (to any doctype) and tries to click the submit button (on top), he gets: "Account 'xxx@yyy.zzz' is not yet activated. Try to login with another account"

Which is misleading... He should get an error that says "You are not authorized to perform this action."
Probably, an extra simple check should be incorporated in websubmit_webinterface for this case...

I've spotted this behavior a few months ago (and notified the list), but I think(?) the issue still stands even for the current version, so I thought it would be better to submit a ticket. Apologies if it was fixed in the meantime :)

FYI, the call is initiated in websubmit_webinterface.py

```
[...]
                if not at_least_one_submission_authorized and submission_exists:

                    if isGuestUser(uid):
                        return redirect_to_url(req, "%s/youraccount/login%s" % (
                            CFG_SITE_SECURE_URL,
                            make_canonical_urlargd({'referer' : CFG_SITE_SECURE_URL + req.unparsed_uri, 'ln' : args['ln']}, {}))
                                            , norobot=True)
                    else:

                        return page_not_authorized(req, "../submit",              # <-- this is executed
                                                   uid=uid,
                                                   navmenuid='submit')
                return home(req,catalogues_text, c,ln)
```

and then page_not_authorized (from webuser) is called:

```
[...]
            if res and res[0][0]:
                if text:
                    body = text
                else:
                    body = "%s %s" % (CFG_WEBACCESS_WARNING_MSGS[9] % cgi.escape(res[0][0]),                # <-- this is executed
                                      ("%s %s" % (CFG_WEBACCESS_MSGS[0] % urllib.quote(referer), CFG_WEBACCESS_MSGS[1])))
[...]
```

then from access_control_config: CFG_WEBACCESS_WARNING_MSGS[9] = Account '%s' is not yet activated.

**Comment by jirikuncar** [comment]
@egabancho @switowski can you confirm if it's still reproducible?

**Comment by switowski** [comment]
@jirikuncar I can't reproduce it on CDS (every logged in user has access to some submissions), I have created a Jira issue for that: https://its.cern.ch/jira/browse/CDS-673

**Comment by tiborsimko** [comment]
Closed by #3729.
